projects/plugins/jetpack/modules/videopress/class.videopress-edit-attachment.php

```diff
@@ -63,86 +63,59 @@ public function configure_meta_boxes( $post_type = 'unknown', $post = null ) {
 	 * @param array      $post
 	 * @param array|null $attachment
 	 *
+	 * Disable phpcs rule for nonce verification since it's already done by Core.
+	 * @phpcs:disable WordPress.Security.NonceVerification
+	 *
 	 * @return array
 	 */
 	public function save_fields( $post, $attachment = null ) {
-		if ( $attachment === null && isset( $_POST['attachment'] ) ) {
+		if ( null === $attachment && isset( $_POST['attachment'] ) ) {
 			$attachment = $_POST['attachment'];
 		}
 
-		if ( ! isset( $attachment['is_videopress_attachment'] ) || $attachment['is_videopress_attachment'] !== 'yes' ) {
+		if ( ! isset( $attachment['is_videopress_attachment'] ) || 'yes' !== $attachment['is_videopress_attachment'] ) {
 			return $post;
 		}
 
-		$post_id = absint( $post['ID'] );
-
-		$meta = wp_get_attachment_metadata( $post_id );
-
 		// If this has not been processed by videopress, we can skip the rest.
 		if ( ! is_videopress_attachment( $post['ID'] ) ) {
+			$post['errors']['videopress']['errors'][] = __( 'The media you are trying to update is not processed by VideoPress.', 'jetpack' );
 			return $post;
 		}
 
-		$values = array();
-
-		// Add the video title & description in, so that we save it properly.
-		if ( isset( $_POST['post_title'] ) ) {
-			$values['title'] = trim( strip_tags( $_POST['post_title'] ) );
-		}
-
-		if ( isset( $_POST['post_excerpt'] ) ) {
-			$values['description'] = trim( strip_tags( $_POST['post_excerpt'] ) );
-		}
-
-		if ( isset( $attachment['rating'] ) ) {
-			$rating = $attachment['rating'];
-
-			if ( ! empty( $rating ) && videopress_is_valid_video_rating( $rating ) ) {
-				$values['rating'] = $rating;
-			}
-		}
-
-		// We set a default here, as if it isn't selected, then we'll turn it off.
-		$values['display_embed'] = 0;
-		if ( isset( $attachment['display_embed'] ) ) {
-			$display_embed = $attachment['display_embed'];
-
-			$values['display_embed'] = 'on' === $display_embed ? 1 : 0;
-		}
-
-		$args = array(
-			'method' => 'POST',
+		$post_title    = isset( $_POST['post_title'] ) ? $_POST['post_title'] : null;
+		$post_excerpt  = isset( $_POST['post_excerpt'] ) ? $_POST['post_excerpt'] : null;
+		$rating        = isset( $attachment['rating'] ) ? $attachment['rating'] : null;
+		$display_embed = isset( $attachment['display_embed'] ) ? $attachment['display_embed'] : 0;
```

Comment:
Shouldn't we use `trim( strip_tags() )` in these `$_POST` parameters?

Reply:
The $_POST parameters are used just to be passed to the existing v2 video endpoint (WPCOM_REST_API_V2_Endpoint_Videos). Not sure though if they're escaped or not.

Reply:
They are escaped here: https://github.com/Automattic/jetpack/blob/d6c9a3c4d95a0660c2954613f0d2c3e6da219715/projects/plugins/jetpack/modules/videopress/class-videopress-attachment-metadata.php#L140